src/main/charts/bamboo/values.yaml

```diff
@@ -340,7 +340,94 @@ ingress:
 # Bamboo configuration 
 #
 bamboo:
-  
+
+  # -- Supply the base URL for the installation.
+  baseUrl:
+
+  # -- Override the server/agent broker URL; this is optional.
+  brokerUrl:
+
+  # The security token with which the agent will authenticate to the Bamboo server
+  # Additional details here:
+  # https://confluence.atlassian.com/bamboo/agent-authentication-289277196.html#Agentauthentication-SecuritytokenverificationSecuritytokenverification
+  #
+  securityToken:
+
+    # -- The name of the K8s Secret that contains the security token. When specified the token 
+    # will overrided the generated one. This secret should also be shared with the agent deployment.
+    # An Example of creating a K8s secret for the secret below:
+    # 'kubectl create secret generic <secret-name> --from-literal=security-token=<security token>'
+    # https://kubernetes.io/docs/concepts/configuration/secret/#opaque-secrets
+    #
+    secretName:
+
+    # -- The key (default `secretKey`) in the Secret used to store the Bamboo shared key.
+    #
+    secretKey: security-token
+
+  # -- Whether to disable agent authentication. Default is false.
+  #
+  disableAgentAuth: false
+
+
+  # -- The license to provide to the Bamboo nodes (optional). If you
+  # have an existing license it can be provided to the server up-front
+  # to skip the configuration screen on first run.
+  #
+  license:
+
+    # -- The secret that contains the license information
+    #
+    secretName:
+
+    # -- The key (default 'licenseKey') in the Secret used to store the license information
+    #
+    secretKey: license
+
+  # -- The admin user configuration and credentials to provide to
+  # Bamboo (optional). If provided this will skip the configuration
+  # screen on first run.
+  #
+  sysadminCredentials:
+
+    # -- The secret that contains the admin user information
+    #
+    secretName:
+
+    # -- The key in the Kubernetes Secret that contains the sysadmin username
+    #
+    usernameSecretKey: username
+    
+    # -- The key in the Kubernetes Secret that contains the sysadmin password
+    #
+    passwordSecretKey: password
+    
+    # -- The key in the Kubernetes Secret that contains the sysadmin display name
+    #
+    displayNameSecretKey: displayName
+    
+    # -- The key in the Kubernetes Secret that contains the sysadmin email address
+    #
+    emailAddressSecretKey: emailAddress
+
+  # -- Bamboo can optionally import an existing exported dataset on
```

Comment:
Can we also include a reference to the official docs on this behaviour i.e.

- https://confluence.atlassian.com/bamboo/exporting-data-for-backup-289277255.html
- https://confluence.atlassian.com/bamboo/importing-data-from-backup-289277260.html